client/components/date-range/style.scss

```diff
@@ -110,19 +110,25 @@ $date-range-mobile-layout-switch: $break-small;
 	justify-content: flex-end;
 	padding: 10px 0 0;
 
-	.button {
+	.components-button {
 		margin-left: 0.5em;
 		margin-right: 0;
+		&.is-compact {
+			height: 28px;
+			font-size: $font-body-extra-small;
+		}
```

Comment:
Can we remove the compact overrides altogether? To me, in this context, it doesn't make a big difference.

Without these styles:
![Screenshot 2025-03-11 at 13 22 16](https://github.com/user-attachments/assets/048b5340-3a4c-4d09-b749-c0a14bdbf332)


With those styles:

![Screenshot 2025-03-11 at 13 22 27](https://github.com/user-attachments/assets/0939b258-a171-4c1e-b36e-408c23f42fea)